src/main/groovy/com/rundeck/plugins/ansible/ansible/AnsibleRunnerBuilder.java

```diff
@@ -811,14 +811,15 @@ public String getBaseDir() {
     }
 
     public String getBinariesFilePath() {
-        String binariesFilePathStr = null;
-        Object binariesFilePath = getjobConf().get(AnsibleDescribable.ANSIBLE_BINARIES_DIR_PATH);
-        if (null != binariesFilePath) {
-            binariesFilePathStr = (String) binariesFilePath;
-            if (binariesFilePathStr.contains("${")) {
-                return DataContextUtils.replaceDataReferences(binariesFilePathStr, getContext().getDataContext());
-            }
-        }
+        String binariesFilePathStr;
+	binariesFilePathStr = PropertyResolver.resolveProperty(
+		AnsibleDescribable.ANSIBLE_BINARIES_DIR_PATH,
+		null,
+		getFrameworkProject(),
+		getFramework(),
+		getNode(),
+		getjobConf()
+		);	
         return binariesFilePathStr;
```

Comment:
to maintain the original behavior, this code should be added
```suggestion
        if (null != binariesFilePathStr) {
            if (binariesFilePathStr.contains("${")) {
                return DataContextUtils.replaceDataReferences(binariesFilePathStr, getContext().getDataContext());
            }
        }
        return binariesFilePathStr;
```